src/frontend/src/catalog.rs

```diff
@@ -154,6 +154,13 @@ impl CatalogManager for FrontendCatalogManager {
         unimplemented!("FrontendCatalogManager does not support register schema")
     }
```

Comment:
Though this PR isn't related to this, Can I also replace `FrontendCatalogManager does not support register schema` with `FrontendCatalogManager does not support registering schema`?

Reply:
> Though this PR isn't related to the `register_schema` method, can I also replace `FrontendCatalogManager does not support register schema` with `FrontendCatalogManager does not support registering schema`?

Cool, please feel free to fix these style issues.